announcement/announcement-api/api/src/bundle/announcement_es.properties

```diff
@@ -25,6 +25,7 @@ java.alert.youacc.pes=No tiene permisos para acceder a los mensajes - {0}
 java.alert.thisis=Este \u00edtem est\u00e1 siendo editado por otro usuario. Por favor, int\u00e9ntelo de nuevo m\u00e1s tarde.
 ### (moot) java.alert.youcreate = you don't have permissions to create this announcement -
 java.alert.youchoosegroup=Debe escoger al menos un grupo.
+java.alert.youchooserole=Debe escoger al menos un rol.
```

Comment:
Put all the new strings together at the end of the file, put a comment for S2U-36.